.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
-    "trailingComma": "none",
-    "tabWidth": 4,
+    "tabWidth": 2,
+    "useTabs": false,
```

Comment:
dette er default, og trenger derfor ikke spesifieres

Reply:
Litt vanskelig å se hvilken du kommenterer på siden det er flere rader som blir quota, men regner med du mener useTabs? Fjerner den isåfall 👍